examples/angular-cli/.storybook/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+
+module.exports = (baseConfig) => {
+  debugger;
+  const originalTsRule = baseConfig.module.rules.find(rule => rule.test.test('text.ts'));
+
+  if (originalTsRule) {
+    originalTsRule.exclude = [/\.stories\.tsx?$/];
+  }
+
+  baseConfig.module.rules.push({
+    test: [
+      /\.stories\.tsx?$/,
+      /index\.ts$/
+    ],
+    loaders: [ ...originalTsRule.loaders, require.resolve('@storybook/addon-storysource/loader') ],
```

Comment:
We can add this loader with `enforce: 'pre'` instead of patching the original rule:
https://webpack.js.org/configuration/module/#rule-enforce